#!/usr/bin/env python
#import roslib;roslib.load_manifest('ipa_loc_feature_slam_development')

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("data_association_threshold_new_feature", double_t, 0, "If the likelihood of feature-observation correspondece is below this threshold for all features, a new feature is added", 0.01, 0.00001, 5)
gen.add("min_existence_estimate_map_feature", int_t, 0, "Candidates with that value will be added to the feature list", 15, 1, 400)
gen.add("best_map_file_name", str_t, 0, "File name for best map", "best_slam_map")
gen.add("best_map_file_path", str_t, 0, "Path to store best map", ".")
exit(gen.generate("ipa_loc_feature_slam_development", "ipa_loc_feature_slam_development", "ipa_loc_feature_slam_development"))
